receiver/snowflakereceiver/documentation.md

```diff
@@ -0,0 +1,84 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# snowflakereceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| snowflake.billing.cloud_service.total | Reported total credits used in the cloud service. | {credits} | Gauge(Double) | <ul> <li>service_type</li> </ul> |
```

Comment:
the documentation is still in the old format. Please make sure to run `make generate` after rebasing